docs/content/SplitPageLayout.mdx

```diff
@@ -0,0 +1,313 @@
+---
+title: SplitPageLayout
+componentId: split_page_layout
+status: Draft
+description: Provides structure for a split page layout, including independent scrolling for the pane and content regions. Useful for responsive list/detail patterns, when an item in the pane updates the page content on selection.
+---
+
+<Note variant="warning">Not implemented yet</Note>
+
+## Examples
+
+### Default
+
+```jsx
+<SplitPageLayout>
+  <SplitPageLayout.Header>
+    <Placeholder label="Header" height={64} />
+  </SplitPageLayout.Header>
+  <SplitPageLayout.Pane>
+    <Placeholder label="Pane" height={120} />
+  </SplitPageLayout.Pane>
+  <SplitPageLayout.Content>
+    <Placeholder label="Content" height={240} />
+  </SplitPageLayout.Content>
+  <SplitPageLayout.Footer>
+    <Placeholder label="Footer" height={64} />
+  </SplitPageLayout.Footer>
+</SplitPageLayout>
+```
+
+### With pane hidden on narrow viewports
+
+```jsx
+<SplitPageLayout>
+  <SplitPageLayout.Header>
+    <Placeholder label="Header" height={64} />
+  </SplitPageLayout.Header>
+  <SplitPageLayout.Pane visible={{narrow: false}}>
```

Comment:
I think there was push back because it matched html spec. In the PVC case `hidden: true` would end up being attributes on the html tag.

Reply:
I also prefer `hidden`, but like @jonrohan mentioned, there's a big confusion in the way PVC is designed, since it handles element attributes and component properties the same way.

Through [`system_arguments`](https://primer.style/view-components/system-arguments#html-attributes), a boolean `hidden` HTML attribute can be passed to any PVC already. Since the visibility prop also needs to accept responsive values (narrow/regular/wide), and doesn't manipulate the hidden attribute directly, we tried to find an alternative.

If you folks think this is not a problem, I'm happy to use `hidden` instead. I'd just like to make sure the API is the same across PRC and PVC.

Reply:
+1 for "hidden". For hidden FormControl labels, we use a prop named `visuallyHidden`.

Reply:
> There would also be no way to actually set `<div hidden class="Component--hiddenWhenNarrow">` hidden attribute on components that support the responsive structure. Which is something that's done for a11y reasons currently

@jonrohan From the MDN docs, it looks like `hidden` as a property and `display: none` as a style are interchangeable. `hidden` can even be overridden by CSS:

> Note: Changing the value of the CSS [display](https://developer.mozilla.org/en-US/docs/Web/CSS/display) property on an element with the hidden attribute overrides the behavior. For instance, elements styled display: flex will be displayed despite the hidden attribute's presence.
> — https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/hidden

For accessibility, the `aria-describedby` [page](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-describedby) doesn't describe _how_ the element could be hidden to remain working as an accessible description.

@alliethu @ichelsea quick a11y question: is the `hidden`[^1] required for accessibility reasons, or would hiding an element with CSS directly (using `display: none`, per instance) work the same way?

[^1]: `hidden`, not `aria-hidden` 😉

Reply:
> in general I'd like system_argument support to be turned off by default, I think moving that way for any new components is the way to go

@jonrohan _yes please_ <3

Reply:
Thank y'all for weighing in. Based on everyone's input so far, I'm going to move forward with the `hidden` prop for this API draft. It seems like we agree that calling the prop `hidden` is better than `visible` from a component API perspective, but we still need to figure out the implementation details. We can discuss those details in future PRs.

Reply:
Chiming in a bit late here, but I'd like to consider using something besides `hidden`. I think we need to be careful of overloading terms in our glossary, and `hidden` is also an HTML attribute. 

While I do generally agree that having system arguments coexist with HTML attributes, in this case I think it's actually a good thing! IMO we should be leaving room for HTML to supercede our component systems as it it ultimately the underlying technology we're building on.